docs/Pitfalls.asciidoc

```diff
@@ -82,7 +82,7 @@ load_templates scripts accordingly.
 
 [id="debugdevelmode"]
 == Steps to debug developer mode setup
-This is basically a checklist to go through in case the developer mode is broken in your setup:
+This is basically a checklist to go through in case the developer mode is broken in your setup (e.g. you are getting the error message `unable to upgrade ws to command server`):
```

Comment:
IMHO the text itself is fine and at the right location. I simply suggest to wrap the lines at 80 characters